pegasus/data/cdo-concepts-csf.csv

```diff
@@ -1,130 +1,130 @@
 id_s!,mainConcept_s*,name_t,age_s,lessonURL_t,lessonPlan_t,lessonVid_t,teacherVid_t,sampleTeachingVid_t,objectives_t,overview_t,additional_t,split_s,part1_s,part2_s,courseNum_s,part3_s,lessonNum_s,part4_s,part5_s,courseLesson_s,answers_t,online_t,ema_t
-diceRace,Algorithms,Dice Race,9+,https://studio.code.org/s/coursee/stage/10/puzzle/1,https://curriculum.code.org/csf/coursee/10/,https://youtu.be/xaW3PAzHxCU,https://youtu.be/xaW3PAzHxCU,," - Decompose large activities into a series of smaller events.
+diceRace,Algorithms,Dice Race,9+,https://studio.code.org/s/coursee/lessons/10/levels/1,https://curriculum.code.org/csf/coursee/10/,https://youtu.be/xaW3PAzHxCU,https://youtu.be/xaW3PAzHxCU,," - Decompose large activities into a series of smaller events.
```

Comment:
I don't remember if the pegasus gsheets have a two-way sync or not, so it's possible these changes would be overwritten on staging

Reply:
Skimming [the code](https://github.com/code-dot-org/code-dot-org/blame/cd774806f02ef271de124459376338ea94b356e6/pegasus/data/csv.rb#L163), it doesn't look like we sync back to drive. I think the safest option here is to remove the .csv files from this PR so you can merge it then we can decide if/how to update those CSVs.

Reply:
I just went into the google sheets and did find and replace there. So I will remove them from this PR and they will get updated when they get scooped.

Reply:
Amazing! You have much more patience than I do :)